driver/fvGFS/fv_nggps_diag.F90

```diff
@@ -74,6 +74,7 @@ module fv_nggps_diags_mod
  use fv_diagnostics_mod, only: range_check, dbzcalc,max_vv,get_vorticity, &
                                max_uh,max_vorticity,bunkers_vector,       &
                                helicity_relative_CAPS,max_vorticity_hy1
+ use module_diag_hailcast
```

Comment:
Would you please list only those variables and subroutines in module_diag_hailcast that are used in this file?